usr/share/rear/restore/NFS_SERVER/default/300_start_nfs_server.sh

```diff
@@ -0,0 +1,35 @@
+# 300_start_nfs_server.sh
+
+# same options works for mountd
+local nfsd_opts="--no-udp --no-nfs-version 3"
+local cpu_cores=$(grep ^cpu\\scores /proc/cpuinfo | uniq | cut -d ':' -f 2 | xargs)
```

Comment:
```suggestion
local cpu_cores=$(nproc)
```

Reply:
in general: https://stackoverflow.com/questions/6481005/how-to-obtain-the-number-of-cpus-cores-in-linux-from-the-command-line

Reply:
In general:
Avoid `local bar="$( COMMAND1 )"`
because 'set -e' will not exit if COMMAND1 exits with a non-zero status.
Also `local baz="$( COMMAND2 )" || Error "COMMAND2 failed"`
will not error out if COMMAND2 failed.
See https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
@jsmeix do you mean to do this in two steps, like this:
```shell
local foo
foo=$( COMMAND ) || Error "some problem"
```

Good catch, I also noticed that once and forgot again. Probably best to simply define all local variables with `local` at the top of the script and then just use them.

Reply:
Yes, with separated commands one gets separated exit codes.

Reply:
yes (sorry, I have known this for some time, but have not spotted it here).

Reply:
No need to be sorry - this is why it helps when more than one have a look.